multi_partner_learning.py

```diff
@@ -270,10 +301,19 @@ def compute_collaborative_round_fedavg(self):
         is_very_first_minibatch = (epoch_index == 0 and minibatch_index == 0)
         x_val, y_val = self.val_data
 
+#TODO: To remove!!!        
+        logger.debug(self.init_model_from)
+        logger.debug(os.path.isfile(self.init_model_from))
+        
+
         # Starting model for each partner is the aggregated model from the previous mini-batch iteration
         if is_very_first_minibatch:  # Except for the very first mini-batch where it is a new model
-            logger.debug(f"(fedavg) Very first minibatch of epoch n°{epoch_index}, init new models for each partner")
-            partners_model_list_for_iteration = self.init_with_new_models()
+            if os.path.isfile(self.init_model_from):
+                partners_model_list_for_iteration = self.init_with_previous_learned_models()
+                logger.debug(f"(fedavg) Very first minibatch of epoch n°{epoch_index}, init models with previous coalition model for each partner")
```

Comment:
2 remarks here:

1. I am not too comfortable with doing the test on the existence of the file. Why not on a `Scenario` attribute corresponding to the option of activating or not this optimization?
1. This should also be applied to `compute_collaborative_round_seqavg()`